packages/jumpstarter-driver-corellium/jumpstarter_driver_corellium/corellium/api.py

```
@@ -0,0 +1,171 @@
+    def get_device(self, model: str) -> Optional[Device]:
+        """
+        Get a device spec from Corellium's list based on the model name.
+
+        A device object is used to create a new virtual instance.
+        """
+        try:
+            res = self.req.get(f'{self.baseurl}/v1/models')
+            data = res.json()
+            res.raise_for_status()
+        except requests.exceptions.RequestException as e:
+            raise CorelliumApiException(data.get('error', str(e)))
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)